pandas/io/gbq.py

```diff
@@ -79,23 +87,158 @@ class InvalidColumnOrder(PandasError, IOError):
     """
     pass
 
+
```

Comment:
why are you adding all of these exceptions? If an exception happens, just have it raise directly. No need to cover over the API. Only really useful exceptions should be done as custom.